.env.example

```diff
@@ -1,6 +1,3 @@
-# Env variables are needed to run tests with API server running
-# This file was copied from stelace-server repo (Jan. 2020)
-
```

Comment:
Why did you remove the date of the last version?

Reply:
My bad, I just copied the new `.env.example` and forgot to keep those lines. I'll fix that.